packages/admin/cms-admin/src/form/file/replace.ts

```diff
@@ -0,0 +1,40 @@
+import { AxiosInstance, AxiosRequestConfig, AxiosResponse, CancelToken } from "axios";
+
+import { GQLLicenseInput } from "../../graphql.generated";
+
+interface UploadFileData {
+    file: File;
+    scope: Record<string, unknown>;
+    folderId?: string;
+    importSourceId?: string;
+    importSourceType?: string;
+    license?: GQLLicenseInput;
+}
+
+export function replace<ResponseData>(
```

Comment:
Isn't this exactly the same as the upload function, except for the different endpoint?

Reply:
ceb76d4afc0ae9e22d34aa4fabbb4069b380c0f3